src/internal/git/pull.go

```diff
@@ -35,14 +36,14 @@ func pull(gitUrl, targetFolder string, spinner *message.Spinner) {
 	gitCred := FindAuthForHost(gitUrl)
 
 	matches := strings.Split(gitUrl, "@")
-	fetchAllTags := len(matches) == 1
+	fetchAll := len(matches) == 1
```

Comment:
This var name is confusing me a bit, especially since we are always using the `!` of its value when checking it.

Thoughts on changing it to something like `fetchSpecificRef := len(matches) == 2`? or `onlyFetchRef`?